### ISSUE TITLE
Documented process for AC_PASSWORD key setup step?

### ISSUE THREAD

**Issue by xrd** [smittytone/scripts#1]
I was reading through your packapp.zsh script and came across this:

```
    echo "This script requires an Apple Developer Account. You will need to set up an app key"
    echo "for the Apple ID linked to your Developer Account, and to have saved this key in your"
    echo "Mac's keychain under the ID 'AC_PASSWORD'. Pass the keychain item's account name to"
```

I am unsure how to setup the app key correctly. Is this documented somewhere that you recommend? I went to the developer.apple.com site and went to "Certificates, IDs & Profiles" and then "Keys". I then created a key for "Account & Organizational Data Sharing" and then downloaded the key. But, I don't see how to import the resulting .p8 file into my keychain, so I believe I didn't complete that step correctly.

I am grateful for this script and if you prefer to do this under a paid consulting engagement, please let me know.

**Comment by smittytone** [comment]
This is what you need: https://support.apple.com/en-us/102654

TL;DR It's an Apple ID app-specific password, not a key from the developer site. You'll get a string code that you can then paste into Keychain Access' **New Password Item** dialog and to which you give the name `AC_PASSWORD`. Set the **Account Name** field to your Apple ID.
